tests/integration/README.md

```diff
@@ -0,0 +1,22 @@
+# Firefox Monitor Integration tests
+
+### Prerequisites
+- Docker
+- Docker-Compose
+- [geckodriver]
+- [npm]
+- [nodejs]
+
+## Firefox Setup
+
+Make sure Firefox is in your system path if you want to watch the tests run inside your browser.
+
+1. Follow setup instructions [here](https://github.com/mozilla/blurts-server#install).
+2. Set appropriate ENV variables, HIBP_API_TOKEN, HIBP_KANON_API_TOKEN, MALINATOR_PASSWORD
+2. Build and start the containers: ```docker-compose -f tests/integration/docker-compose.yml up --build -d```.
+3. Run the tests: ```npm run test:integration```.
+   They can also be run headlessly: ```npm run test:integration-headless```, and inside docker: ```npm run test:integration-docker```.
+
+[geckodriver]: https://github.com/mozilla/geckodriver/releases
```

Comment:
Ah, interesting! looks like geckodriver is on npm too: https://www.npmjs.com/package/geckodriver (by our very own vladikoff!)

Reply:
Should I link that instead?

Reply:
No, sorry, that was just me thinking out loud and bookmarking it for future reference. It'd only make sense to switch if we added it as a project dependency in package.json and got rid of the explicit dependency of having it preinstalled. But I'd have to audit the npm install to see how big that bundle is. In fact, I can do that right now.

Looks like running <kbd>npm i geckodriver -D</kbd> in a fresh directory adds about 9.7 MB of node_modules goodness, including the following files in _./node_modules/geckodriver/_:

```
-rwxr-xr-x   5.0M 10 Oct 15:53 geckodriver
-rw-r--r--   1.9M  5 Mar 10:26 geckodriver.tar.gz
```

But let's just leave this as-is for now and can refactor in the future after this lands.

Reply:
Good info to have though!